src/Tests/Aggregations/Bucket/Terms/TermsAggregationUsageTests.cs

```diff
@@ -105,56 +108,4 @@ protected override void ExpectResponse(ISearchResponse<Project> response)
 			states.Meta["foo"].Should().Be("bar");
 		}
 	}
-
-	public class NumericTermsAggregationUsageTests : AggregationUsageTestBase
```

Comment:
For documentation, I think it may be better for this to remain in the TermsAggregationUsageTests file so that it will appear on https://www.elastic.co/guide/en/elasticsearch/client/net-api/5.x/terms-aggregation-usage.html (The file also needs asciidoc floating headers in it to explain the different examples).